app/controllers/hosts_controller.rb

```diff
@@ -193,10 +193,7 @@ def externalNodes
 
     begin
       respond_to do |format|
-        # don't break lines in yaml to support Ruby < 1.9.3
-        # Remove the HashesWithIndifferentAccess using 'deep_stringify_keys',
-        # then we turn it into YAML
-        host_info_yaml = @host.info.deep_stringify_keys.to_yaml(:line_width => -1)
+        host_info_yaml = @host.info.to_yaml(:line_width => -1)
```

Comment:
You mentioned this didn't have a significant impact, so i think we should skip it - i believe it is needed to make sure that hash keys are properly rendered in the enc

Reply:
just a note of caution, if people ENC output will be different, that should be a breaking API change, please note changes in ENC output effects system configurations! afaiu deep merge etc are all there for a reason?

Reply:
Yes, I was planning to drop it but I wanted to bring it to reviewers attention. Thing is - why we do have this? There is a note this is for 1.9.x version which is unsupported now. I also did compare my output with and without this method and I see no difference. So why is that?

Reply:
If it's a minor performance factor, perhaps it makes sense to send it as a separate PR so you don't have multiple discussions going on at once? https://theforeman.org/2019/03/merge-time-review-complexity.html

Reply:
Someone explain to me why this method should be there? In YAML there is no reason between symbol and string, is there?

```
 [3] pry(main)> {test: 1}.to_yaml
=> "---\n:test: 1\n"
[4] pry(main)> {'test': 1}.to_yaml
=> "---\n:test: 1\n"
```

> just a note of caution

I understand, test coverage of this action is rather poor.

Reply:
```ruby
{test: 1}.keys.first.class
=> Symbol

{:test => 1}.keys.first.class
=> Symbol

{"test" => 1}.keys.first.class
=> String
```

Reply:
I have just found we actually have tests for this controller, I was searching in wrong directory. Now I understand it, I will put this call back:

```
Rails logs for test_0016_externalNodes should render YAML hashes correctly FAILURE:
HostsControllerTest#test_0016_externalNodes should render YAML hashes correctly = 0.19 s = F


Failure:
HostsControllerTest#test_0016_externalNodes should render YAML hashes correctly [test/controllers/hosts_controller_test.rb:257]:
--- expected
+++ actual
@@ -81,9 +81,11 @@
 classes:
   dhcp:
     bootfiles:
-    - name: foo
+    - !ruby/hash:ActiveSupport::HashWithIndifferentAccess
+      name: foo
       mount_point: \"/bar\"
-    - name: john
+    - !ruby/hash:ActiveSupport::HashWithIndifferentAccess
+      name: john
       mount_point: \"/doe\"
 environment: production
```

Reply:
Yeah thanks Adam, I forgot that YAML supports these nasty things. Okay, makes sense, I am putting this back.